bcos-boostssl/websocket/WsService.cpp

```diff
@@ -296,6 +296,7 @@ bool WsService::registerMsgHandler(uint32_t _msgType, MsgHandler _msgHandler)
 
 std::shared_ptr<WsSession> WsService::newSession(std::shared_ptr<WsStream> _stream)
 {
+    _stream->setReadMaxMsg(m_config->maxReadMsgSize());
```

Comment:
Pls make sure the option effective here

Reply:
the option had been proved be effective by testing message size using sample in test directory.